Samples~/ClientSimIssue3RespawnTest/ClientSimIssue3RespawnTest.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System.Collections;
+using System.IO;
+using NUnit.Framework;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.TestTools;
+using UnityEngine.UI;
+using VRC.SDK3.ClientSim;
+using VRC.SDK3.ClientSim.Tests.WorldTests;
+using VRC.Udon.Common;
+
+namespace ClientSimTest.Tests.WorldTests
+{
+    public class ClientSimIssue3RespawnTest : ClientSimWorldTestBase
+    {
+        private static readonly string _sceneName = "ClientSimIssue3RespawnTest";
+        private static readonly string _samplesDirectory = Path.Combine("Assets", "Samples", "VRChat Client Simulator");
+        private static bool _sceneExists;
+        
+        protected override ClientSimSettings GetTestSettings()
+        {
+            return new ClientSimSettings
+            {
+                enableClientSim = true,
+                initializationDelay = 0,
```

Comment:
I found that leaving this at zero will not catch Udon errors that run in start. If you add a small delay here, it will catch them. Obviously not needed for this test, but might be useful in the future. 0.5f seemed to work for me, but I'm not sure if this is machine dependent.

Reply:
Thanks, updated to 0.5 now and added a comment to this effect.